### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by VishnuSanal**

### CODE REVIEW

**Comment by TranceLove** [comment]
Need resolve conflict here @VishnuSanal

**Comment by VishnuSanal** [comment]
@VishalNehra @TranceLove PTAL 🙂

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,42 +1,55 @@
...
+Happy to see you in here! Amaze was created with a vision to provide all the basic and advanced features a file manager in Android is supposed to have, with a friendly material design. As with any other open source project, contributions are the key to achieve this goal of ours. :)
 
 Contributions are always welcome!
 
-To start contributing, we assume you know how to use git and write and debug Android apps - don't expect we can teach you ;)
+> ### Imposter's syndrome disclaimer: We want your help. No, really! Any contribution counts, no matter how small it be!
+
+# How to contribute?
+
+- **Translation**: We use [Transfeix](https://www.transifex.com/amaze/amaze-file-manager/) for our translations, feel free to contribute translations there.
```
Comment (TranceLove) [inline CONTRIBUTING.md]: Transifex

**Comment by TranceLove** [comment]
@VishnuSanal A little typo before approval 🙂

**Review by TranceLove — APPROVED** [review]

**Review by VishalNehra — APPROVED** [review]

**Pull request merged**